client/src/main/java/com/netflix/conductor/client/http/ClientBase.java

```diff
@@ -24,6 +24,7 @@
 import com.netflix.conductor.common.run.ExternalStorageLocation;
 import com.netflix.conductor.common.utils.ExternalPayloadStorage;
 import com.netflix.conductor.common.validation.ErrorResponse;
+import com.netflix.conductor.core.utils.BulkResponse;
```

Comment:
A better approach here would be to move the `BulkResponse` object to the `common` module.